qpc/utils.py

```diff
@@ -408,3 +408,12 @@ def create_tar_buffer(files_data):
             tar_file.addfile(tarinfo=info, fileobj=file_buffer)
     tar_buffer.seek(0)
     return tar_buffer.getvalue()
+
+
+def get_settings():
```

Comment:
Let's not stick this in util as a function.  So I think the settings should be it's own module (e.g. release_settings.py).  I think it should load the json on startup.  Additionally, have accessor methods.  Let's chat tomorrow, but I think it will be a little different from this.